README.md

```diff
@@ -25,6 +25,12 @@ $ ./cove/manage.py runserver
 
 CoVE is based on Django and be deployed using the deployment mechanisms as defined in the [Django docs](https://docs.djangoproject.com/en/3.1/howto/deployment/)
 
+### Run tests
+
+Set up [chromedriver](<https://chromedriver.chromium.org/getting-started>), then run:
+
```

Comment:
I added the chromedriver autoinstaller, so the tests only actually require chrome and `requirements_dev.txt" to be installed https://github.com/ThreeSixtyGiving/dataquality/blob/main/requirements_dev.in#L16

Reply:
True! I have removed it.